app/schema.py

```
@@ -0,0 +1,35 @@
+# If you want to keep your other schemas from your original code, place them here:
+class QuerySchema(BaseModel):
+    query: str
+    backend: str = "open-deepresearch"
+    planner_model: str = "gpt-4o-mini"
+    writer_model: str = "gpt-4o-mini"
+    report_structure: Optional[str] = (
+        "Comprehensive analysis with key findings, details, and implications"
+    )
+
+
+class QueriesSchema(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)